### PR TITLE
Load env image frontend

### PR DESCRIPTION
**Pull request by joshcwinton**
This PR is a temporary solution to rendering the environment on the chat page

When the environment is updated, the backend renders a new image, which is fetched from the frontend

Issue: Closes #

### Frontend:
- use image from backend in `Environment.jsx` instead of placeholder
- re-fetch image with `updateEnvironment` whenever a message is sent
### Backend:
- clear old image when server restarts to start with blank image

### CODE REVIEW

File: backend/render.py [hunk 1]
```diff
@@ -1,11 +1,11 @@
 from PIL import Image, ImageDraw, ImageFilter
```
Comment (JasperCheung) [inline backend/render.py:1]: Can you add PIL to the `requirements.txt`? or whatever is the right package.

**Review by JasperCheung — CHANGES_REQUESTED** [review]
I'm also getting a `FileNotFoundError: [Errno 2] No such file or directory: 'images/env_image.png'` When trying to run `python routes.py` for the first time.


I think its because you didn't commit a `/images` dir. So we either gotta make initialize one with a empty file and then .gitignore it or do it by code and create one. I like the the first option.

File: backend/render.py (hunk 1 above)
Comment (joshcwinton) [reply backend/render.py:1]: added

**Comment by joshcwinton** [comment]
@JasperCheung added changes 👍

File: .gitignore [hunk 2]
```diff
@@ -158,3 +158,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# Images directort
```
Comment (JasperCheung) [inline .gitignore:162]: lmao

**Review by JasperCheung — APPROVED** [review]
lgtm. 🧯

File: .gitignore (hunk 2 above)
Comment (joshcwinton) [reply .gitignore:162]: ffffffffffffffff 🤦

**Pull request merged**